### DISCUSSION TITLE
Asset Inventory: Capturing perils to which each asset is exposed - For each hazard - Capture mitigation measures that have been taken, and when, and their likely benefit - T #53

### DISCUSSION THREAD

**Discussion by sreyac** [CriticalAssetManagement/CAMS#126] [General]
Need clarification on this and discussion around implementation

**Comment by rpwilliams1958** [comment]
For the MVP this could be a simple listing of the hazards (wind, coastal flooding, wildfire etc) known to affect the area in which the asset is located.  Mitigation measures would be a text field; As for likely benefit, the issue is do the mitigation measures save to deal with the most severe hazard that has been projected in a professional risk study?  Answers can be "Yes" [ie a risk analysis has been carried out] , "No" and "Don't Know".
